src/template/fake-template/Content/build.blackfox.fsx

```diff
@@ -0,0 +1,34 @@
+//#if (dependencies == "inline" && dsl == "fake")
```

Comment:
I think you can get rid of those conditions, because you only copy the file when necessary, right?